src/pages/settings/following.vue

```diff
@@ -104,6 +104,23 @@ export default {
       }));
     },
   },
+  watch: {
+    walletHasLoggedIn: {
+      immediate: true,
+      async handler(hasLoggedIn) {
+        // Only fetch followees in client side
+        if (process.client && hasLoggedIn) {
```

Comment:
Does this cause ssr outsync error? can we just call this in mounted?

Reply:
I don't see any SSR out sync

Reply:
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/17264716/220703105-5b12c48c-624b-45fa-99aa-134a127411b8.png">

https://user-images.githubusercontent.com/17264716/220703471-0db037a7-002c-4899-a6ba-e1c00eefe397.mov